ansible-ipi-install/inventory/hosts.sample

```diff
@@ -114,7 +118,11 @@ prov_ip=172.22.0.3
 domain=""
 # Name of the cluster, i.e. openshift
 cluster=""
-# The public CIDR address, i.e. 10.1.1.0/21
+# The public CIDR address, i.e. 10.1.1.0/21 in list format, containing only one entry for
+# IPv6 only or IPv4 only baremetal network. List contains two elements, with the first entry
+# being the IPv4 range in the case of dual-stack. Example for dual-stack is
+# extcidrnet=["192.168.216.0/21", "fd01:1101::/64"]. extcidr is a list with atleast one element
+# and not more than two elements.
```

Comment:
# The public CIDR address, i.e. 10.1.1.0/21 containing only one entry for
# IPv6 or IPv4 for the baremetal network. When using dual-stack it must be in list format,
# and the list should contain two elements, with the first entry
# being the IPv4 range and the second for IPv6.
#
# Example for dual-stack is # extcidrnet=["192.168.216.0/21", "fd01:1101::/64"].
# Example for single-stack would be extcidrnet="192.168.216.0/21".

Reply:
Hi, @iranzo . So you are seeing let extcidrnet be a string for ipv4/ipv6 only and a list for dual-stack? I appreciate the suggestion but I think there are some problems
1. I think it is inherently better to not change "variable type". I think a list should always be a list and a string a strin, otherwise it introduces unnecessary complexity which needs multiple checks/validations
2. Also, it introduces problems in other parts of the inventory if a user decides to use `extcidrnet` variable to populate other fields. For example, `dnsvip="{{ extcidrnet[0] | next_nth_usable(2) }}"` in inventory/hosts will always work regardless of IPv6/IPv4/dual-stack. However, if we alternate in between list/string some if this jinja2 templating based varibale assignment in the inventory also needs to change.

Reply:
Right now, extcidrnet is a string, this PR will break all users as it will change that in a list.

If we focus on users, CI's, etc to continue being able to use what they had without introducing breacking ranges, that's the way to do it.

If the user makes customizations for the inventory, etc it's not something we're doing, but converting it in a list yes.

Reply:
@iranzo maybe there is some other solution? To me it seems like have the same var be a list and also a string, could end up being confusing. Open to other ideas....